odoo/addons/base/models/ir_actions.py

```diff
@@ -559,6 +559,7 @@ def _default_update_path(self):
              "- 'Execute Code': a block of Python code that will be executed\n"
              "- 'Send Webhook Notification': send a POST request to an external system, also known as a Webhook\n"
              "- 'Multi Actions': define an action that triggers several other server actions\n")
+    allowed_states = fields.Json(string='Allowed states', compute="_compute_allowed_states")
```

Comment:
we only need that for documents account no ?
can we move the code of the widget there ? and that field too
(and with xpath, add the widget)